src/platform/intel/cavs/lib/pm_runtime.c

```diff
@@ -42,6 +42,8 @@ DECLARE_SOF_UUID("power", power_uuid, 0x76cc9773, 0x440c, 0x4df9,
 
 DECLARE_TR_CTX(power_tr, SOF_UUID(power_uuid), LOG_LEVEL_INFO);
 
+static int dsp_sref[PLATFORM_CORE_COUNT];	/**< simple ref counter */
```

Comment:
Part of the `struct cavs_pm_runtime_data`?

Reply:
moved to struct cavs_pm_runtime_data